test/nebulex/caching_test.exs

```diff
@@ -10,8 +12,18 @@ defmodule Nebulex.CachingTest do
 
   defmodule Meta do
     defstruct [:id, :count]
+    @type t :: %__MODULE__{}
   end
 
+  # defmodule TestKeyGenerator do
```

Comment:
should remove this commented out block

Reply:
Indeed, it is still WIP.

Reply:
definitely. Just trying to provide helpful eyes :)